tdrs-backend/tdpservice/scheduling/parser_task.py

```diff
@@ -18,4 +21,8 @@ def parse(data_file_id):
 
     logger.info(f"DataFile parsing started for file {data_file.filename}")
     errors = parse_datafile(data_file)
-    logger.info(f"DataFile parsing finished with {len(errors)} errors: {errors}")
+    dfs = DataFileSummary.objects.create(datafile=data_file, status=DataFileSummary.Status.PENDING)
+    dfs.status = dfs.get_status(errors=errors)
+    dfs.case_aggregates = case_aggregates_by_month(data_file)
```

Comment:
```suggestion
    dfs.status = dfs.get_status(errors=errors)
    if dfs.status == DataFileSummary.Status.REJECTED:
      # error-handle
    else:  
      dfs.case_aggregates = case_aggregates_by_month(data_file)
```

Reply:
Write specific test-case for this with pre-parser and ensuring we generate the "N/A" for field.